helix-front/server/controllers/d.ts

```diff
@@ -5,6 +5,7 @@ export interface HelixUserRequest extends Request {
 }
 
 interface HelixSession {
+  identityToken: any;
```

Comment:
I would generally agree, however since this token format is user-customizable, this really is `any` from helix-front's point of view.  helix-front doesn't know what Identity Source it will be used with, or what type that Identity Source's identityToken will be.